.github/workflows/development-ci.yml

```diff
@@ -0,0 +1,37 @@
+name: Development to Development-CI Sync
+
+on:
+  push:
+    branches:
+      - development
+  workflow_dispatch:
+
+permissions:
+  contents: write
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    name: Sync changes from development to development-ci
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Run merge
+        run: |
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          git config --global user.name "github-actions[bot]"
```

Comment:
This seems like a hack/work around. Are you sure this is the correct way to do this?

Reply:
yes